src/CommandLine.csproj

```diff
@@ -1,36 +1,46 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <!-- Import the NuSpec info about the package -->
-  <Import Project="..\pkg\nuspec.props" />
 
   <PropertyGroup>
-    <OutputType>library</OutputType>
-    <TargetFrameworks>netstandard1.5</TargetFrameworks>
-    <Version>1.4.0</Version>
-    <AssemblyVersion>1.4.0.0</AssemblyVersion>
-    <FileVersion>1.4.0.0</FileVersion>
+    <TargetFrameworks>netstandard1.5;</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net45;</TargetFrameworks>
+    <DebugType>pdbonly</DebugType>
```

Comment:
I assume this is about getting the code coverage data, right?

Reply:
Yes